client/my-sites/people/people-section-nav/index.jsx

```diff
@@ -74,7 +74,9 @@ class PeopleSectionNav extends Component {
 				id: 'viewers',
 			},
 			{
-				title: translate( 'Invites' ),
+				title: isEnabled( 'subscriber-importer' )
+					? translate( 'User Invites' )
+					: translate( 'Invites' ),
```

Comment:
**comment**: I'm not 100% familiar with this code, so please feel free to correct me if I'm wrong here!

This code checks the presence of a feature flag located under `config/`, and sets the tab title depending on the presence of the feature flag. 

If what I said above is true, then I see a possible issue with the E2E tests, since the changes to `calypso-e2e` in this PR defines `User Invites` to be the only permissible value. So if for whatever reason the feature flag is turned off and never shipped to production, we might run into situations where the E2E will begin failing.

I did come up with a possible solution which is a bit outside the scope of this PR, so if my statement above is true, I will submit a followup PR to make it easier to deal with situations such as this.

Reply:
Hey @worldomonation, you are right.

Theoretically, there is a possibility that our feature never sees production, and potentially e2e tests can start failing again.

My thinking was only a happy path, but as you suggest, it will be nice to provide handling of this situation in a follow-up PR. Thanks for that!

Reply:
I'm totally ok with that, so I'm closing this PR.

Thanks @worldomonation

Reply:
So as a follow-up, it looks like I had already accounted for this in my design of the framework-level `clickTab` implementation. See [this file](https://github.com/Automattic/wp-calypso/blob/965f1779322a4d368c1e25ace34d1c2119f2d254/packages/calypso-e2e/src/element-helper.ts#L8-L10) and note the selector ultimately uses a `:has-text` which instructs Playwright to perform fuzzy matching.

So no changes are needed, as the current definition of "Invite" tab in [PeoplePage](https://github.com/Automattic/wp-calypso/blob/trunk/packages/calypso-e2e/src/lib/pages/people-page.ts#L5) would match on `User Invites`.